web-marketplace/src/pages/Dashboard/MyProjects/MyProjects.constants.ts

```diff
@@ -3,3 +3,5 @@ import { msg } from '@lingui/macro';
 export const DRAFT_ID = 'draft';
 export const CREATE_POST = msg`+ create post`;
 export const CREATE_POST_DISABLED_TOOLTIP = msg`You cannot make posts because either the project is still a draft or it doesn't have assigned a location.`;
+export const MY_PROJECTS_BUTTON_TEXT = '+ create project';
+export const MY_PROJECTS_EMPTY_TITLE = 'You have not created any projects yet';
```

Comment:
Add translations

Reply:
It was already translated in another PR.